docs/tutorial.md

```diff
@@ -0,0 +1,312 @@
+into [Allotrope Simple Model][allotrope] (ASM) format,
+i.e.,
+to take the (often messy) output of laboratory tools
+and turn it into something that is well structured and has a well-defined semantics
+so that other pieces of software can use it.
+
+ASM is a complex format,
+but that complexity is necessary to represent the wide variety of inputs it must handle.
+For example,
+some plate readers produce several readings per well,
+so the format needs a way to represent a one-to-many relationship.
+Similarly,
+other devices might include timestamps,
+sample numbers,
+operator IDs,
+and other information,
+so ASM as a whole has to accommodate these
+even for devices that don't generate that information.
+
+`allotropy` represents the structures defined by ASM as Python classes
+that can be converted to and from JSON.
+It does not convert those nested structures into flat formats
+like Pandas dataframes
+because different applications will want to extract different pieces of information
+from the overall representation.
+However,
+producing dataframes and other representations is straightforward,
+and is covered in this tutorial.
+
+## A Simple Input File
+
+The input files for this tutorial come from an old Weyland-Yutani plate reader,
+which produces CSV files like this:
+
+```csv
+Weyland-Yutani 470 1251,,,,
+Recorded,2023-10-23:17:54:48,,,
+,,,,
+,A,B,C,D
+1,4.87,3.45,0.92,4.71
+2,3.29,0.99,0.76,3.30
+3,2.93,0.57,3.52,3.73
+4,1.94,4.46,2.25,4.05
+```
+
+Line by line,
+the structure of this file is:
+
+-   One line with the device model number (Weyland-Yutani 470) and serial number (1251),
+    all in one column.
+-   Another line with the word `Recorded` in the first column
+    and an ISO-formatted timestamp in the second column.
+-   A "blank" line.
+    Note that this line, and all others,
+    actually contains enough commas to delimit five columns
+    even if values are absent.
+-   Five lines of plate data.
+    The first line has a blank followed by column titles A-D;
+    the remaining four rows have row numbers followed by readings.
+
+If the operator pushes the right buttons,
+the Weyland-Yutani 470 can also produce files like this:
+
+```csv
+Weyland-Yutani 470 1384,,,,
+Recorded,2023-10-26:11:15:40,,,
+,,,,
+,A,B,C,D
+1,3.61,1.45,2.23,3.08
+2,4.10,4.58,3.60,1.67
+3,3.27,1.40,4.99,2.47
+4,2.78,0.72,4.00,0.49
+,,,,
+Checksum,b855,,,
+```
+
+This file has a different machine serial number (1384 instead of 1251),
+but more importantly,
+the plate readings are followed by another "blank" line consisting of nothing but commas
+and then a checksum of the readings.
+Unfortunately,
+the documentation available to us does not specify how this checksum is calculated.
+
+## Defining Files
+
+After forking the [`allotropy` repository`][allotropy_repo]
+and creating a new branch for our work,
+we make a new directory `./src/allotropy/parsers/example_weyland_yutani/` for our work
+and create three files in that directory:
```

Comment:
this sentence seems to have been a bit misformatted, would you be able to fix this? 

<img width="1015" alt="image" src="https://github.com/Benchling-Open-Source/allotropy/assets/42189754/15f710ef-57fc-4b73-9e51-5fbc2e0261a7">

Reply:
fixed - thank you